.github/workflow_scripts/test_text.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ex
+
+source $(dirname "$0")/env_setup.sh
+
+setup_build_env
+export CUDA_VISIBLE_DEVICES=0
+install_core_all_tests
+install_features
```

Comment:
is `install_features` already in `install_multimodal`?

Reply:
We need to install separately as we are building from source.